docs/swagger.yaml

```diff
@@ -470,31 +467,31 @@ components:
       x-examples:
         BottomLineExample:
           value:
-            totalAmount: 6000
-            totalTax: 1200
-            totalDiscount: 2500
             currency: NOK
             tipAmount: 0
             giftCardAmount: 0
             terminalId: vipps_pos_122
       properties:
         totalAmount:
           type: integer
-          description: 'The total amount of the order, including all taxes, discounts and shipping. Can be negative'
+          description: 'Deprecated, sum will be calculated based on the sum of the orderLines'
           format: int64
           nullable: true
+          deprecated: true
         totalTax:
           type: integer
-          description: Total tax for the order
+          description: 'Deprecated, tax will be calculated based on the sum of the orderLines'
```

Comment:
Deprecated, tax will be calculated based on the sum of the totalTaxAmount field on each orderLine